core/graph/src/main/java/uk/gov/gchq/gaffer/graph/hook/NamedViewResolver.java

```diff
@@ -38,20 +39,21 @@
  * A {@link GraphHook} to resolve {@link NamedView}s.
  */
 @JsonPropertyOrder(alphabetic = true)
-public class NamedViewResolver implements GraphHook {
+public class NamedViewResolver implements GetFromCacheHook {
     private final NamedViewCache cache;
 
     @JsonCreator
-    public NamedViewResolver(@JsonProperty("suffixCacheName") final String suffixCacheName) {
-        cache = new NamedViewCache(suffixCacheName);
+    public NamedViewResolver(@JsonProperty("suffixNamedViewCacheName") final String suffixNamedViewCacheName) {
```

Comment:
This may have already been mentioned, but if this is in the docs then I think we can get away with this technical breaking change being rolled out quickly after 2.0

Reply:
I've got this covered in the docs. You're right that this is breaking.